geode-redis/src/integrationTest/java/org/apache/geode/redis/StringsIntegrationTest.java

```diff
@@ -82,7 +82,6 @@ public static void setUp() {
   @After
   public void flushAll() {
     jedis.flushAll();
```

Comment:
what is wrong with doing jedis2.flushAll here? In other places you added a jedis2 call but in this one (and in the Hashes test) you removed it.

Reply:
it should be removed from everywhere.  flushall is a server action that wipes all the data, not a client one, so only one client needs to do it. These integration tests are running against a single server.